### PR TITLE
Cluster-Autoscaler: upcoming nodes in ClusterStateRegistry

### PR DESCRIPTION
**Pull request by mwielgus**
Ref: #2228 #2229

### CODE REVIEW

File: cluster-autoscaler/clusterstate/clusterstate.go [hunk 1]
```diff
@@ -332,9 +333,31 @@ func isNodeNotStarted(node *apiv1.Node) bool {
 	for _, condition := range node.Status.Conditions {
 		if condition.Type == apiv1.NodeReady &&
 			condition.Status == apiv1.ConditionFalse &&
-			condition.LastTransitionTime.Time.Sub(node.CreationTimestamp.Time) < time.Second {
+			condition.LastTransitionTime.Time.Sub(node.CreationTimestamp.Time) < 5*time.Second {
```
Comment (jszczepkowski) [inline cluster-autoscaler/clusterstate/clusterstate.go]: Shouldn't 5*timeSecond be a const with an appropriate name?

File: cluster-autoscaler/clusterstate/clusterstate_test.go [hunk 2]
```diff
@@ -203,3 +203,43 @@ func TestRegisterScaleDown(t *testing.T) {
...
+	now := time.Now()
+	ng1_1 := BuildTestNode("ng1-1", 1000, 1000)
+	setReadyState(ng1_1, true, now.Add(-time.Minute))
+
+	ng2_1 := BuildTestNode("ng2-1", 1000, 1000)
+	setReadyState(ng2_1, false, now.Add(-time.Minute))
+
+	ng3_1 := BuildTestNode("ng3-1", 1000, 1000)
+	setReadyState(ng3_1, false, now.Add(-time.Minute))
+	ng3_1.CreationTimestamp = metav1.Time{Time: now.Add(-time.Minute)}
```
Comment (jszczepkowski) [inline cluster-autoscaler/clusterstate/clusterstate_test.go]: Can you add comment, what is happening here?

**Comment by jszczepkowski** [comment]
Just two small comments, LGTM after they are addressed.

File: cluster-autoscaler/clusterstate/clusterstate.go (hunk 1 above)
Comment (mwielgus) [inline cluster-autoscaler/clusterstate/clusterstate.go]: Done.

File: cluster-autoscaler/clusterstate/clusterstate_test.go (hunk 2 above)
Comment (mwielgus) [inline cluster-autoscaler/clusterstate/clusterstate_test.go]: Done.

**Pull request merged**